### ISSUE TITLE
[BUG] Module loading error: define, module: app/rspamd

### ISSUE THREAD

**Issue by latchee** [rspamd/rspamd#4821]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Read about bug reporting in general: https://rspamd.com/doc/faq.html#how-to-report-bugs-found-in-rspamd
    * Enabled relevant debugging logs: https://rspamd.com/doc/faq.html#how-to-debug-some-module-in-rspamd 
    * Checked the FAQs about Core files in case of fatal crash: https://rspamd.com/doc/faq.html#how-to-figure-out-why-rspamd-process-crashed
    * Tried ASAN package and obtained the ASAN report (if possible): https://rspamd.com/doc/faq.html#asan-builds 
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Arspamd
    * Checked that there is not already an experimental package or master branch

**Describe the bug**
Cannot login in Rspamd webui anymore. When I access the webui, error below appears:

> Module loading error: define, module: app/rspamd

Then nothing, the password prompt does not show up.

### Steps to Reproduce

1. Access webui via the server's IP (http://192.168.1.xxx)
2. Web UI loads up as usually
3. Red bar at the top shows error "Module loading error: define, module: app/rspamd" with only the option to reload.

**Expected behavior**
Error bar should not show up and password prompt should instead pop up

### Versions
rspamd --version
Rspamd daemon version 3.8.1

CPU architecture x86_64; features: avx2, avx, sse2, sse3, ssse3, sse4.1, sse4.2, rdrand
Hyperscan enabled: TRUE
Jemalloc enabled: TRUE
LuaJIT enabled: TRUE (LuaJIT version: LuaJIT 2.1.1700008891)
ASAN enabled: FALSE
BLAS enabled: FALSE
Fasttext enabled: FALSE

OS: Ubuntu 22:04
I am using Rspamd's repository from https://rspamd.com/downloads.html (not Ubuntu's repo)

**Comment by moisseev** [comment]
Which browser version you are using?
Is any error logged in the web developer tools console?

**Comment by latchee** [comment]
Hi @moisseev ,
I am using Firefox 122.
In the page source, I can see this error:

> <div class="notification-area">
> 	<noscript>
>     	<div class="alert alert-error"><strong>Please enable JavaScript</strong></div>
>     </noscript>
> </div>

However, I verified, javascript is enabled. I also disabled "Enhanced tracking protection" in Firefox and it did not resolve the issue.

One error shows up in the browser console:

> Error: Could not find any MenuItem with id: autofill-identity 2 undefined

However, I then installed Chrome (yikes ;) ) and it works! The red bar error with "Module loading error: define, module: app/rspamd" does not pop up and password prompt pops up. I can then type it in a login.
I suppose this is a bug with Firefox and will try again on a later release. In the meantime, I will be using Chrome.

Thank you for your help :)

**Comment by moisseev** [comment]
It looks like a problem with some Firefox add-on.
Would you temporarily disable extensions in the Firefox Add-ons manager?

**Comment by latchee** [comment]
Hi Moisseev,

Indeed you are right, I had disabled my addons but not restart firefox. After a restart, and all addons all disabled, the page loaded properly.
Issue appeared to be with the addon ublock origin, I think. Even though I had set an exception for Rspam's webui, it blocked stuff from it. A restart must have forced it to refresh its cache or something like that. As it is working fine now.
Thank you very much.
